.eslintrc.js

```diff
@@ -0,0 +1,16 @@
+module.exports = {
+    "env": {
+        "browser": true,
+        "es2021": true,
+        "node": true
+    },
+    "extends": "eslint:recommended",
+    "parserOptions": {
+        "ecmaVersion": 13,
+        "sourceType": "module"
+    },
+    "rules": {
```

Comment:
I think we shouldn't add any custom rules to eslint. What was the reason for adding the two below?

Reply:
I put in  ecmaVersion  because I thought it was default. I can remove it but if I try to remove         "sourceType": "module" , Github action checks will get error

Reply:
I was specifically referring to the content of "rules":

```      "camelcase": "error",
      "no-unused-vars": "warn"
```

Reply:
Okay well , camelcase: This rule looks for any underscores ( _ ) located within the source code. This rule only flags definitions and assignments but not function calls.

no-unused-vars : Github actions check fails because in index.js 'equalityOp' is assigned a value but never used. 
So I configured it to warn instead of throw error so that the checks pass.